### PR TITLE
Switch to the conda-forge-manager user's GH_TOKEN on Travis

### PR DESCRIPTION
**Pull request by jakirkham**
This, as far as I understand it, is still incomplete. The goal here is to mostly address item 1 in issue ( https://github.com/conda-forge/conda-forge.github.io/issues/172 ). The rest of it would be dealt with by PR ( https://github.com/conda-forge/staged-recipes/pull/733 ).

In other words, if staged-recipes uses a separate bot and token, it should not suffer from rate limiting when the feedstocks/team update script does. As a result, people should be able to continue merging and creating new feedstocks unhindered. By doing this alone, it does not guarantee that people will be added to teams in a timely manner. Instead that would require PR ( https://github.com/conda-forge/staged-recipes/pull/733 ). While this does a very negligible job at addressing the underlying rate limiting problem, it does decouple the rate limiting problem that occurs with the feedstock/team update script from work being done at staged-recipes. We intend to further decouple these two so the full feedstock addition process can happen here without be tied the feedstock/team update script at all. This itself should be a huge win.

cc @pelson

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @pelson

**Comment by jakirkham** [comment]
@pelson, can you please let me know if this is good to go or if there are more changes needed? I just copied the string that you gave me. If it is good to go, please feel free to remove "WIP" and merge.

**Comment by pelson** [comment]
I guess there is a reference to conda-forge-admin which also needs updating. That could possibly be it...

**Comment by jakirkham** [comment]
Good eye. I'll fix that real quick.

**Comment by jakirkham** [comment]
There were a couple other `conda-forge-admin` references that I missed. I have gone ahead and updated them. Running `git grep conda-forge-admin` comes back clean now.

File: .travis.yml [hunk 1]
```diff
@@ -8,10 +8,10 @@ env:
...
+        # Add the GH_TOKEN environment variable. (This is the "conda-forge.github.io" token for the conda-forge-manager user.)
+        - secure: "QVcAIw1ezgkfZ70nNQBsr03V9tzXv0Hb+mw759gNo1CBbydYTO5EutWUt/Nks6E12/SkX5I3CmlAHZC74dQXVzQVyU/+hdc0El2N51gkvJ02dua7701ClqOHbkZSPGxE2j/S7ERzJKdtH97N0EGILP9wnadQJs5kpF9MvB1oWOQmDmsGMZ6GGlj2FMfL/TmtNR4ADo8rxr/ylw1kX8anfrXCv7FShdviugpsjSpAQLShKbdeDPdhvrD8W0TlpUhgdgSU2kPXd6gfGKuety1EWh56MgarHg+unntvB8R2W0O07RgOiRa/Mplmeng18dUCJdT1i3rczB2BkeyeFeCihtXqa0KDx7bX4z1X+4YFrBNl/bnDeVijaBtvkK3T0lluVnmVWolRzO9Tc2LozzAUEhDih4QFL6QtxQP2NkRFY2xEcj0JWbBNvDCik7hakiuxBSSbzGr1/3zw2sRh35/6qB6Y7mE6a5Pk+MS1ExS3Z8YUp4Q9l589DeyBCNE6HQHwEdvzEPUW74Lm566lzhsI8D4IuVYZDZLi/HHPzfPav4WWj5jiwlLx9XOkfPgz6ezqLoThkKSp5JmpXxxWzrckOMzFOyUqqD9h7De80uJqM/SW926fCr4RrrExcimblNV0ro9SSHUUvw+ZEFM51P7Z2m6ATBMO7/lVQtpj4XCbs6M="
 
-        # Add the CIRCLE_TOKEN environment variable. (This will need re-generating in the near future using the conda-forge-admin user)
+        # Add the CIRCLE_TOKEN environment variable. (This will need re-generating in the near future using the conda-forge-manager user)
```
Comment (pelson) [inline .travis.yml]: Hmmm. I'm trying to remember if this comment is factual, or whether it is a copy&pasta problem. If the former, we need to keep the conda-forge-admin words, if the latter, we should remove the comment.

Comment (jakirkham) [inline .travis.yml]: I see. Sorry I can't help clarify this. Please just let me know what you decide.

Comment (pelson) [inline .travis.yml]: It needs to remain conda-forge-admin. Thanks @jakirkham.

Comment (jakirkham) [inline .travis.yml]: Ok, will update. Thanks @pelson. Is the value of this token still ok?

Comment (pelson) [inline .travis.yml]: Yes, it is the token that is used to register CircleCI, and comes from their own API (and web interface).

Comment (jakirkham) [inline .travis.yml]: That's what I figured. Just wanted to make sure our changed user didn't have other effects.

**Comment by jakirkham** [comment]
The CircleCI token comment has been reverted. Thanks for catching that, @pelson. 😄

**Comment by jakirkham** [comment]
I dropped the WIP as I'm guessing we are done? Please feel free to merge when you are happy or provide more errata @pelson.

**Comment by jakirkham** [comment]
Thanks @pelson. Looks things have slowed a bit on Travis CI. So, will have to take stock tomorrow.

**Comment by jakirkham** [comment]
So, it ran, but we have an issue @pelson. Appears `conda-forge-manager` is not an admin. Also, looks like it is not in the org. I'll invite the bot, but I'm guessing you will have to confirm it can join. Here's the error message and [log line]( https://travis-ci.org/conda-forge/staged-recipes/jobs/141399078#L320 ). I'll revert this for now and will open a PR so we can try later.

```
You need admin access to the organization before adding a repository to it.
```

**Comment by jakirkham** [comment]
Reverting for now with PR ( https://github.com/conda-forge/staged-recipes/pull/926 ).

**Comment by jakirkham** [comment]
This is being reproposed in PR ( https://github.com/conda-forge/staged-recipes/pull/927 ).

**Comment by pelson** [comment]
Yep, thanks @jakirkham.

**Pull request merged**